src/Features/CSharp/Portable/Completion/CompletionProviders/CompletionUtilities.cs

```diff
@@ -68,6 +69,21 @@ internal static bool IsTriggerCharacter(SourceText text, int characterPosition,
             return false;
         }
 
+        internal static bool IsTriggerCharacterOrArgumentListCharacter(SourceText text, int characterPosition, OptionSet options)
+        {
+            return IsTriggerCharacter(text, characterPosition, options) || IsArgumentListCharacter(text, characterPosition, options);
+        }
+
+        internal static bool IsArgumentListCharacter(SourceText text, int characterPosition, OptionSet options)
+        {
+            return IsArgumentListCharacter(text[characterPosition]);
+        }
+
+        internal static bool IsArgumentListCharacter(char ch)
+        {
+            return ch == '(' || ch == '[' || ch == ' ';
```

Comment:
not sure how `[` will work, given that you check for ArgumentList in other locations.

Reply:
I fixed the other checks.